test/unit/evault/modules/Governance/reserves.t.sol

```diff
@@ -0,0 +1,150 @@
+
+        startHoax(user1);
+        eTST.deposit(50e18, user1);
+        startHoax(user2);
+        eTST.deposit(10e18, user2);
+
+        assertEq(eTST.totalAssets(), 60e18);
+        assertEq(eTST.accumulatedFees(), 0);
+
+        startHoax(user3);
+        evc.enableController(user3, address(eTST));
+        eTST.borrow(5e18, user3);
+
+        skip(30.5 * 86400);
+
+        assertApproxEqAbs(eTST.debtOf(user3), 5.041955e18, 0.000001e18);
+        assertApproxEqAbs(eTST.accumulatedFeesAssets(), 0.003146e18, 0.000001e18);
+
+        assertApproxEqAbs(eTST.maxWithdraw(user1), 50.03234e18, 0.000001e18);
+        assertApproxEqAbs(eTST.maxWithdraw(user2), 10.0064681e18, 0.000001e18);
+
+        // Some more interest earned:
+        skip(90 * 86400);
+
+        assertApproxEqAbs(eTST.debtOf(user3), 5.167823e18, 0.000001e18);
+        assertApproxEqAbs(eTST.accumulatedFeesAssets(), 0.012586e18, 0.000001e18);
+        assertApproxEqAbs(eTST.accumulatedFees(), 0.012554e18, 0.000001e18);
+
+        // Now let's try to withdraw some reserves:
+        startHoax(address(this));
+        eTST.setFeeReceiver(user4);
+        startHoax(admin);
+        protocolConfig.setFeeReceiver(user5);
+
+        eTST.convertFees();
+
+        assertApproxEqAbs(eTST.balanceOf(user5), protocolShare(0.012554e18), 0.000001e18);
+        assertApproxEqAbs(eTST.balanceOf(user4), riskManagerShare(0.012554e18), 0.000001e18);
+        assertEq(eTST.accumulatedFees(), 0);
+
+        // More starts to accrue now:
+        skip(15);
+
+        assertApproxEqAbs(eTST.accumulatedFees(), 0.000000018e18, 0.000000001e18);
+    }
+
+    //withdraw without any deposit is a no-op as amount is zero
+    function test_convertFees_withoutAnyDeposit() public {
+        startHoax(admin);
+        protocolConfig.setVaultInterestFeeRange(address(eTST), true, 0.075e4, 1e4);
+
+        startHoax(address(this));
+        eTST.setInterestFee(0.075e4);
+        eTST.setInterestRateModel(address(new IRMTestFixed()));
+
+        assertEq(eTST.accumulatedFees(), 0);
+        assertEq(eTST.accumulatedFeesAssets(), 0);
+        assertEq(eTST.totalSupply(), 0);
+        assertEq(eTST.totalAssets(), 0);
+
+        skip(30.5 * 86400);
+
+        startHoax(address(this));
+        eTST.setFeeReceiver(user4);
+        startHoax(admin);
+        protocolConfig.setFeeReceiver(user5);
+
+        vm.expectEmit();
+        emit Events.VaultStatus(0, 0, 0, 0, eTST.interestAccumulator(), eTST.interestRate(), block.timestamp);
+        eTST.convertFees();
+
+        assertEq(eTST.balanceOf(user5), 0);
+        assertEq(eTST.accumulatedFees(), 0);
+    }
+
+    function test_setInterestFee_outOfBounds() public {
+        startHoax(address(this));
+        vm.expectRevert(Errors.E_BadFee.selector);
+        eTST.setInterestFee(1.01e4);
+    }
+
+    function protocolShare(uint256 fees) internal pure returns (uint256) {
+        return fees * 0.1e18 / 1e18;
```

Comment:
0.1e18 is the PROTOCOL_FEE_SHARE. We could add a test that would change this value on the vault and make sure convertFees is using the new split.